.changeset/shaggy-wasps-care.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/cloudflare': minor
+---
+
+Adds support for external Cloudflare Image Resizing Service. See Cloudflare docs for more information about Pricing and Features: https://developers.cloudflare.com/images/image-resizing/
```

Comment:
Should we also link to the announcement post, that their pricing will change starting Nov 15?

https://blog.cloudflare.com/merging-images-and-image-resizing/

Reply:
cc @withastro/maintainers-docs